staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/internal/manager.go

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package internal
+
+import (
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+// RemoveObjectManager removes the manager field from the object
+// so we don't persist it.
+func RemoveObjectManager(obj runtime.Object) {
```

Comment:
is this meant for testing only?

Reply:
see https://github.com/kubernetes/kubernetes/pull/75917/files#diff-438c18fccd3899026928547522f80743R218
and https://github.com/kubernetes/kubernetes/pull/75917/files#diff-438c18fccd3899026928547522f80743R141